### DISCUSSION TITLE
Aliexpress LED strip controller device (C01Z) supported?

### DISCUSSION THREAD

**Discussion by ferhatkaratasfk** [Koenkk/zigbee2mqtt#25317] [Devices]
Hello,
Does somebody know if this Aliexpress LED controller device is supported by zigbee2mqtt (see picture below)? I could not find it in the database but it seems to work Tuya. I do not know if this means that it automatically works with zigbee2mqtt/Home Assistant as well. Thanks for your help.
![Schermafbeelding 2024-12-25 094047](https://github.com/user-attachments/assets/d7b31504-a226-4128-b6c4-4c1c0da2f6ed)

**Comment by RoyMai88693** [comment]
Did you find out?

**Comment by ferhatkaratasfk** [comment]
I did not buy as I was not sure. I decided to Shelly RGBW controller instead and have it integrated with Home Assistant.

Verstuurd vanaf mijn iPhone

Op 23 apr 2025 om 10:38 heeft RoyMai88693 ***@***.***> het volgende geschreven:

﻿

Did you find out?
